lib/token.rb

```diff
@@ -0,0 +1,13 @@
+module Token
+    def player_x
+        :X
```

Comment:
No need to change this right now, but what if I wanted to be able to choose my symbol on the TTT board?

Reply:
This part wouldn't change, as it's not tied to a string / rendered board -- this is just for our internal data.  I would create a new setting type class, get the input and store in game state. then refer to it when I create a board output. 

Would changing the name to... maybe Symbols or something help?

Reply:
I do like Symbols.  There's a good amount of valid names here, probably, and Token/Tokens is definitely one.  Naming is rough.